requirements/constraints.txt

```diff
@@ -27,23 +27,15 @@ django-storages<1.9
 # for them.
 edx-enterprise==3.46.6
 
-# Newer versions need a more recent version of python-dateutil
-freezegun==0.3.12
-
 # oauthlib>3.0.1 causes test failures ( also remove the django-oauth-toolkit constraint when this is fixed )
 oauthlib==3.0.1
 
 # django-auth-toolkit==1.3.3 requires oauthlib>=3.1.0 which is pinned because of test failures
 django-oauth-toolkit<=1.3.2
 
-# Upgrading to 2.5.3 on 2020-01-03 triggered "'tzlocal' object has no attribute '_std_offset'" errors in production
-python-dateutil==2.4.0
-# matplotlib>=3.4.0 requires python-dateutil>=2.7
-matplotlib<3.4.0
-# pandas>0.22.0 requires python-dateutil>=2.5.0
-pandas==0.22.0
-# networkx>=2.6 requires pandas>=1.1
-networkx<2.6
+# numpy>=1.17 is required for matplotlib>=3.5.1
+# and numpy>=1.17.0 caused failures in importing numpy in code-jail environment.
+matplotlib==3.4.0
```

Comment:
This looks like it would upgrade matplotlib from 3.3.4 to 3.4.0, but I don't see a corresponding change in `py38.txt`.  And I'd rather not risk upgrading that in codejail at the same time as we attempt the python-dateutil upgrade, so can we shift this back to `matplotlib<3.4.0`?

Reply:
Done. Will be upgraded in separate PR if required.